src/App.css

```diff
@@ -49,3 +53,89 @@ body {
   font-size: large;
   font-weight: 500;
 }
+
+.single-article-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 10px;
+  grid-auto-rows: minmax(100px, auto);
+}
+
+.single-article {
+  text-align: center;
+  grid-column: 2 / 3;
+  grid-row: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 20px;
+  padding: 20px;
+  margin-top: 20px;
+  background-color: rgba(0, 0, 0, 0.226);
+  box-shadow: 1px 1px 10px 1px black, 0em 0 0.4em black;
+}
+
+.loading {
+  text-align: center;
+  grid-column: 2 / 3;
+  grid-row: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 20px;
+  padding: 20px;
+  margin-top: 20px;
+  background-color: rgba(0, 0, 0, 0.226);
+}
+.nav {
+  display: flex;
+  justify-content: space-around;
+  border-bottom: 2px solid rgb(26, 20, 20);
+  flex-wrap: wrap;
+}
+.nav-list {
+  display: flex;
+}
+
+h1 {
+  color: white;
+  text-decoration: underline rgb(25, 118, 210) 6px;
+  text-underline-offset: 2px;
+  text-shadow: 10px, 10px, 10px, 10px black;
+  cursor: pointer;
+  font-size: xx-large;
+}
+
+.nav > h4 {
+  text-shadow: 10px, 10px, 10px, 10px black;
+  cursor: pointer;
+  font-size: x-large;
+}
+
+.nav-icons {
+  text-shadow: 10px, 10px, 10px, 10px black;
+  cursor: pointer;
+  font-size: x-large;
+  display: flex;
+  gap: 10px;
+  list-style: none;
+  padding-top: 15px;
+}
+
+.single-comment {
+  border: 4px solid #adadad65;
+
+  background-color: rgba(54, 54, 54, 0.39);
+  height: 100px;
+  align-items: center;
+  padding: 20px;
+}
+
+.comments-title {
+  color: white;
+  text-decoration: underline rgb(25, 118, 210) 6px;
+  text-underline-offset: 2px;
+  text-shadow: 10px, 10px, 10px, 10px black;
+  cursor: pointer;
+  font-size: xx-large;
+}
```

Comment:
Love the styling Tom this site looks so swish
This is minor but I am struggling to see the border between the articles and the background in the list of articles though